shark-ai/pyproject.toml

```diff
@@ -30,6 +30,9 @@ Repository = "https://github.com/nod-ai/SHARK-Platform"
 onnx = [
   "iree-base-compiler[onnx]",
 ]
+apps = [
+  "shortfin[apps]",
+]
```

Comment:
We also considered other names for `extras` (docs here: https://packaging.python.org/en/latest/specifications/dependency-specifiers/#extras).

* `sd` or `sdxl`: these are very specific, but we don't want to have an extra for every model (sd3, llama2, llama3, flux, etc.), unless there is good reason for that. The `transformers`, `dataclasses-json`, and `pillow` dependencies here aren't specific to SDXL either - they are generic ML / data / image libraries
* `examples` or `demos`: these suggest a lower support level than `apps`
* `apps`: this matches the code path and will let us eventually include other deps for other apps. We can later add more specific extras if `apps` gets too large